tools/fiptool/fiptool.c

```diff
@@ -20,8 +20,8 @@
 
 #include <openssl/sha.h>
 
+#include <firmware_image_package.h>
```

Comment:
Can you move this include to be the alphabetical order, at line 12? 
I know that the include order without your changes is wrong, but as you're changing this file, could you also put the includes at lines 7,8 and 21 into the right alphabetical order too?

Also, please update the copyright date to 2016-2017